assets/component-facets.css

```
@@ -582,7 +590,7 @@ span.active-facets__button-inner {
   font-size: 1rem;
   min-height: 0;
   min-width: 0;
-  padding: 0.5rem 1rem;
+  padding: 0.25rem 1rem;
```

```suggestion
  padding: 0.25rem 1.25rem;
```

Also suggest to add this for a better horizontal white space